src/storage.ts

```diff
@@ -15,3 +17,26 @@ export async function setPersistedProjects(projects: Project[]) {
     projects: JSON.stringify(projects),
   });
 }
+
+export async function getPersistedEntries(): Promise<Entry[]> {
+  const temp = await chrome.storage.sync.get(null);
```

Comment:
Is this line necessary?

Reply:
I don't think so, I copy and pasted it from the other storage function